docs/tutorials/integrations/kserve/abn-grpc.md

```diff
@@ -26,113 +26,101 @@ This tutorial describes how to do A/B testing of a backend ML model hosted on [K
+  modelFormat: sklearn
+  runtime: kserve-mlserver
+  protocolVersion: v2
+  ports:
+  - containerPort: 9000
+    name: h2c
+    protocol: TCP
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    storageUri: "gs://seldon-models/sklearn/mms/lr_model"
 EOF
 ```
 
-The `initialize` action (with strategy `none`) creates a routemap that only defines the resources that make up each version of the application. In this case, a single `InferenceService`.
-Since no version-specific naming is provided, the primary version is expected to be named `backend-0` and any candidate version `backend-1`. Iter8 uses this information to identify when any of the versions of the application are available. It can then respond appropriately to `Lookup()` requests. 
-
 ## Generate load
 
 In one shell, port-forward requests to the frontend component:
     ```shell
     kubectl port-forward service/frontend 8090:8090
     ```
+
 In another shell, run a script to generate load from multiple users:
     ```shell
     curl -s https://raw.githubusercontent.com/iter8-tools/docs/v0.17.3/samples/abn-sample/generate_load.sh | sh -s --
     ```
  
 ## Deploy candidate
 
-Deploy the candidate version of the backend model:
+Release a candidate version of the backend model:
 
 ```shell
-cat <<EOF | kubectl apply -f -
-apiVersion: "serving.kserve.io/v1beta1"
-kind: "InferenceService"
-metadata:
-  name: "backend-1"
-  labels:
-    app.kubernetes.io/name: backend
-    app.kubernetes.io/version: v1
-    iter8.tools/watch: "true"
-spec:
-  predictor:
-    model:
-      modelFormat:
-        name: sklearn
-      runtime: kserve-mlserver
-      protocolVersion: v2
-      storageUri: "gs://seldon-models/sklearn/mms/lr_model"
-      ports:
-      - containerPort: 9000
-        name: h2c
-        protocol: TCP
+cat <<EOF | helm upgrade --install backend $CHARTS/release -f -
+environment: kserve
+application: 
+  metadata:
+    labels:
+      app.kubernetes.io/version: backend
+  modelFormat: sklearn
+  runtime: kserve-mlserver
+  protocolVersion: v2
+  ports:
+  - containerPort: 9000
+    name: h2c
+    protocol: TCP
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    storageUri: "gs://seldon-models/sklearn/mms/lr_model"
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    storageUri: "gs://seldon-models/sklearn/mms/lr_model"
 EOF
 ```
 
 ??? note "About the candidate"
-    In this tutorial, the model source (field `spec.predictor.model.storageUri`) is the same as for the primary version of the model. In a real example, this would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
+    In this tutorial, the model source (field `storageUri`) is the same as for the primary version of the model. In a real example, this would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
```

Comment:
Similar comment as before. Mention that more than the version label can be used.